metadata/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+class Character(models.model):
+
+    """ Definition of Character Data """
+    name = models.charfield(max_length = 50)
```

Comment:
소문자로 써도 마이그레이션 잘되나요? 
```suggestion
    name = models.CharField(max_length = 50)
```